syft/pkg/url.go

```diff
@@ -11,13 +11,13 @@ import (
 )
 
 const (
-	purlArchQualifier   = "arch"
-	purlDistroQualifier = "distro"
-	purlEpochQualifier  = "epoch"
-	purlVCSURLQualifier = "vcs_url"
+	ArchQualifier   = "arch"
```

Comment:
note on naming: renaming these removes the context that it is specifically for pURLs and not for just syft packages.

Reply:
I thought about this, but the fact these are all labeled as "Qualifier" seemed to be sufficient? Maybe we should name these either `PURLQualifier<Name>` or `<Name>PURLQualifier`? They do need to be exported.